src/person/serializer.py

```diff
@@ -15,7 +15,7 @@ class Meta:
         )
         read_only_fields = fields
 
-    def get_access_token(self):
+    def get_access_token(self, instance):
```

Comment:
why do you need the instance? you're not using it

Reply:
instance gets passed in when i call authenticateserializer

Reply:
k